cluster/v1alpha1/types.go

```diff
@@ -233,6 +233,13 @@ type PlacementSpec struct {
 	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
 	// +optional
 	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+
+	// PrioritizerConfigs defines the configurations of the prioritizers.
+	// Default configuration is used if it is not specified, in which Steady and Balance prioritizers have the weight of 1
```

Comment:
If I specific `{ResourceRatioCPU}`, what is the resulting weight of `Steady`?

Reply:
it (Steady) will still use the default weight of 1 if it is not specified here.

Reply:
after discussed with @elgnay and @haoqing0110. We think If we specific `{ResourceRatioCPU}` but not specify `Steady`, it means the `Steady` is disabled. 

If the whole PrioritizerConfigs is not set, it will use the default values which is `Steady` & `Balance` =1, and others = 0